tests/python/test_job_api.py

```
@@ -0,0 +1,146 @@
+    def test_no_top_level_header(self) -> None:
+        self.assertFalse(hasattr(rookie_cookies, "header"))
+        self.assertNotIn("header", rookie_cookies.__all__)
+
+    def test_as_list_schema_and_iter(self) -> None:
+        with _synthetic_home() as home:
+            _seed_chrome(home)
+            result = rookie_cookies.read(browser="chrome", include_expired=True)
+        rows = result.as_list()
+        self.assertGreater(len(result), 0)
+        self.assertTrue(result)
+        for row, iterated in zip(rows, result):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)